test/crs/conversions.jl

```diff
@@ -1706,6 +1706,58 @@
       @inferred convert(LatLon, c2)
     end
 
+    @testset "LatLon <> PolarStereographicB" begin
+      # forward tested against Proj.Transformation("""
+      # proj=pipeline
+      # step proj=axisswap order=2,1
+      # step proj=unitconvert xy_in=deg xy_out=rad
+      # step proj=stere lat_0=-90 lat_ts=-71 lon_0=0 x_0=0 y_0=0 ellps=WGS84
+      # """)
+      # inverse tested against Proj.Transformation("""
+      # proj=pipeline
+      # step inv 
+      # proj=stere lat_0=-90 lat_ts=-71 lon_0=0 x_0=0 y_0=0 ellps=WGS84
+      # step proj=unitconvert xy_in=rad xy_out=deg
+      # step proj=axisswap order=2,1
+      # """)
+      c1 = LatLon(T(-80), T(0))
+      c2 = convert(PolarStereographicB{-71°,WGS84Latest}, c1)
+      @test allapprox(c2, PolarStereographicB{-71°,WGS84Latest}(T(0.0), T(1.0891794556261837e6)))
+      c3 = convert(LatLon, c2)
+      @test allapprox(c3, c1)
+
+      c1 = LatLon(T(-80), T(90))
+      c2 = convert(PolarStereographicB{-71°,WGS84Latest}, c1)
+      @test allapprox(c2, PolarStereographicB{-71°,WGS84Latest}(T(1.0891794556261837e6), T(0.0)))
+      c3 = convert(LatLon, c2)
+      @test allapprox(c3, c1)
+
+      c1 = LatLon(T(-80), T(-90))
+      c2 = convert(PolarStereographicB{-71°,WGS84Latest}, c1)
+      @test allapprox(c2, PolarStereographicB{-71°,WGS84Latest}(T(-1.0891794556261837e6), T(0.0)))
+      c3 = convert(LatLon, c2)
+      @test allapprox(c3, c1)
+
+      c1 = LatLon(T(-80), T(180))
+      c2 = convert(PolarStereographicB{-71°,WGS84Latest}, c1)
+      @test allapprox(c2, PolarStereographicB{-71°,WGS84Latest}(T(0.0), T(-1.0891794556261837e6)))
+      c3 = convert(LatLon, c2)
+      @test allapprox(c3, c1)
+
+      c1 = LatLon(T(-89), T(10))
+      c2 = convert(PolarStereographicB{-71°,WGS84Latest}, c1)
+      @test allapprox(c2, PolarStereographicB{-71°,WGS84Latest}(T(18867.758184287333), T(107004.37396749199)))
+      c3 = convert(LatLon, c2)
+      @test allapprox(c3, c1)
+
+
```

Comment:
```suggestion
```

You can press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>I</kbd> on VSCode to format the code according to the project standards. This extra line will be erased for example.